tests/pbft_manager_test.cpp

```diff
@@ -601,36 +600,9 @@ TEST_F(PbftManagerTest, pbft_manager_run_multi_nodes) {
 
   std::cout << "Checking all nodes see transaction from node 1 to node 2..." << std::endl;
 
-  bool checkpoint_passed = false;
-  for (auto _(0); _ < 120; ++_) {
-    checkpoint_passed = true;
-    // test timeout is 60 seconds
-    for (size_t i(0); i < nodes.size(); ++i) {
-      if (nodes[i]->getDB()->getNumTransactionExecuted() == 0) {
-        checkpoint_passed = false;
-      }
-    }
-    if (checkpoint_passed) break;
-    taraxa::thisThreadSleepForMilliSeconds(500);
-  }
-
-  if (checkpoint_passed == false) {
-    for (size_t i(0); i < nodes.size(); ++i) {
-      ASSERT_EQ(nodes[i]->getDB()->getNumTransactionExecuted(), 1);
-    }
-  }
-
-  uint64_t pbft_chain_size = 1;
-  for (size_t i(0); i < nodes.size(); ++i) {
-    EXPECT_EQ(nodes[i]->getFinalChain()->last_block_number(), pbft_chain_size);
-  }
-
-  for (size_t i(0); i < nodes.size(); ++i) {
-    std::cout << "Checking account balances on node " << i << " ..." << std::endl;
-    EXPECT_EQ(nodes[i]->getFinalChain()->getBalance(node1_addr).first, node1_genesis_bal - 100);
-    EXPECT_EQ(nodes[i]->getFinalChain()->getBalance(node2_addr).first, 100);
-    EXPECT_EQ(nodes[i]->getFinalChain()->getBalance(node3_addr).first, 0);
```

Comment:
I see that balances are checked inside wait_for_balances, but I dont see any "testing conditions" like these there  ?

Reply:
If we are waiting for that balances then after that wait it would be equal to that one's we expects. So no need to add this check one more time. Or am I getting your comment wrong?